search/search_quality/assessment_tool/edits.hpp

```diff
@@ -13,32 +13,42 @@
 class Edits
 {
 public:
+  using Relevance = search::Sample::Result::Relevance;
+
+  struct Entry
+  {
+    Entry() = default;
+
+    Relevance m_curr = Relevance::Irrelevant;
```

Comment:
+1, from my point of view datatype + name suggests the correct meaning of the variable.